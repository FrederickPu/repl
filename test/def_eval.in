{"cmd": "def f := 37"}

{"cmd": "#check (rfl : f = 37)"}
