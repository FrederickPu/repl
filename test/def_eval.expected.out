{"results": [{"source": "def f :=\n  37", "env": 0}]}

{"results":
 [{"source": "#check (rfl : f = 37)",
   "messages":
   [{"severity": "info",
     "pos": {"line": 1, "column": 0},
     "endPos": {"line": 1, "column": 6},
     "data": "rfl : f = f"}],
   "env": 1}]}

